### DISCUSSION TITLE
Can we get more docs on inscribing on the client side?

### DISCUSSION THREAD

**Discussion by WhitoshiBlackamoto** [omnisat/lasereyes-mono#282] [Ideas]
We need more info about how to inscribe on the client side. This is important because many inscribing services are doing it wrong and expensive.. Can we get some examples how to do it?

**Comment by hathbanger** [comment]
great idea @WhitoshiBlackamoto! and so sorry we missed, thanks @ufe-pr for bringing it to our attention.
